fagiClient/src/com/fagi/network/InputHandler.java

```diff
@@ -61,7 +62,7 @@ public void run() {
                     }
                     running = false;
                     ChatManager.closeCommunication();
-                    ChatManager.getApplication().showLoginScreen();
+                    Platform.runLater(() -> ChatManager.getApplication().showLoginScreen());
```

Comment:
If `Platform.runLater()` is removed, it will cause an issue on `MasterLogin` line 64. Wouldn't it make sense to have the `Platform.runLater` call on as little code as possible. In this way we keep the main thread doing as little possible.